hummingbot/strategy/amm_arb/amm_arb_config_map.py

```diff
@@ -82,6 +82,12 @@ def order_amount_prompt() -> str:
         prompt_on_new=True,
         validator=market_2_validator,
         on_validated=market_2_on_validated),
+    "pool_id": ConfigVar(
+        key="pool_id",
+        prompt="Specify poolId to interract with on the DEX connector >>> ",
+        prompt_on_new=True,
```

Comment:
Change to prompt_on_new=`False`

I think users should not need to enter a poolID. However, if they do enter a poolID, it should be used instead of the trading pair.

Reply:
Noted.

Reply:
@fengtality @nikspz This is done